allenbolanos/dotTheory-FE
# FE-102: Front-End Formations

# Related Tickets
https://codeandtheory.atlassian.net/browse/FE-530

# Description
Creates the Comments component.
Creates 5 copies of the detail page.
Creates a browse component on the navbar.

# Validation
- open the FE-102 Front-End Formations folder
- run any html file on the articles folder on a live server
- comments section should be found at the bottom part of the page
- browse dropdown component can be found at the navbar
